wordpress/plugins/region-report/single-region_report.php

```diff
@@ -368,7 +370,137 @@ function addToList(list, text) {
 
             <section>
                 <h3>Research Effort</h3>
+                
                 <div class="region-report-research-effort">
+                    <div id="region-report-research-effort-<?php the_ID(); ?>-1"></div>
+                    <div id="region-report-research-effort-<?php the_ID(); ?>-2"></div>
+                </div>
+                <script>
+                     postElement.addEventListener(
+                        "researchEffort",
+                        e => {
+                            const values = [];
+                            e.detail.forEach(e => {
+                                values.push({
+                                    year: e.year,
```

Comment:
You have two distinct bindings for "e" here, which might get confusing (sometimes it's a useful "trick" that makes code read clearer, but I worry here that it's hard to keep track of which e is what)